### ISSUE TITLE
Build failure due to "missing" file

### ISSUE THREAD

**Issue by etherbrat** [Muxelmann/linux-dss#1]
After typing "make", the following error  message:
  (1002) Target OS: Linux for x86-64
  (3104) Compiling electricdss/Source/LazDSS/DirectDLL/OpenDSSDirect.lpr
  Fatal: (2012) Cannot open file "OpenDSSDirect.lpr"
  Fatal: (1018) Compilation aborted
  Error: /usr/bin/ppcx64 returned an error exitcode
  makefile:45: recipe for target 'all' failed

The file in question appears to be present in various subdirectories:
  find . -name OpenDSSDirect.lpr
  ./electricdss/Version7/Source/DDLL/OpenDSSDirect.lpr
  ./electricdss/Version7/Source/Deprecated_LazDSS/DirectDLL/OpenDSSDirect.lpr
  ./electricdss/Version8/Source/CMD_Lazz/DirectDLL/OpenDSSDirect.lpr

**Comment by M-A-abeer-Sohail** [comment]
Yes I am able to replicate the same bug. Looks like no one got to solving it?

By the way, my original goal was to use some of my MATLAB code that used OpenDSS COM objects, within an Ubuntu environment. I believe you might have had the same motivation to use this repo. Were you able to make it work?

**Comment by ashokkushh** [comment]
do you get the solution?
